setup.py

```diff
@@ -5,7 +5,7 @@
 
 from setuptools import setup
 
-install_requires = ["psutil", "progressbar2", "numpy", "ansys.grpc.dpf>=0.2.3"]
+install_requires = ["psutil", "progressbar2", "numpy", "ansys.grpc.dpf>=0.2.3", "protobuf<=3.20.1"]
```

Comment:
to be merged?

Reply:
@cbellot000 Indeed it has been merged in the meantime. A merge with the current master has been added.